README.md

```diff
@@ -121,12 +132,18 @@ These are the current pipelines running on the repo:
 
 # Testing
 
+The tests within /tests are designed to connect to a real database to run tests that use the DB layer, details below. This takes a long time to setup on a test run, so if your tests don't need a DB put them in /tests_no_db. This folder doesn't pickup /tests/conftest.py so therefore doesn't do the DB setup work and these tests can run in isolation much faster.
+
 ## Seed Postgres DB with mock data
 
 `invoke seed_dev_db`
 
 Running the above command will prompt you to enter the number applications, funds & rounds you would like to create as mock data within the database.
 
+This will also work for the DB within the docker runner. Find the ID of the docker container running assessment-store (`docker ps`) then execute:
+
+        docker exec -it <container_id> invoke seed_dev_db
```

Comment:
I think it's a bit confusing the mix between the docker and local instructions but I think maybe we can pick this up as part of a bigger bit of work looking at consistent use of databases and how to run the tests, so outside the scopr of this ticket.